client/src/components/PreviewPopUp.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogActions from "@mui/material/DialogActions";
+
+export default function PreviewPopUp(props: {
+  title: string;
+  body: string;
+  img: string;
+  tags: string;
+  eventCapacity: string;
+  location: string;
+  openPopup: boolean;
+  handleClose: any;
```

Comment:
This should be a function type. For example, `(a: string) => void` would be the annotation for a function which takes a string (one param) and returns nothing.